src/main/scala/ai/starlake/utils/SQLUtils.scala

```diff
@@ -50,6 +56,81 @@ object SQLUtils {
     ctes.map(_.replaceAll("`", ""))
   }
 
+  def extractColumnNames(sql: String): List[String] = {
```

Comment:
Does jSqlParser infer * select or not ?

```
with
  my_table as (select col1, col2, col3),
  my_table2 as (select col1, col4)
  select my_table.* except(col2), myTable2.col4 from my_table inner join my_table2 using(col1)
```

expected output is col1, col3, col4.